test/fixtures/add-identifier-with-top-level-import-paths/code.js

```diff
@@ -1,13 +1,12 @@
-import styledA from '@xstyled/styled-components/test'
-import styledB from '@xstyled/styled-components-test'
-import styledC from '@xstyled/styled-components'
+import styled from '@xstyled/styled-components'
+import unstyled from '@xstyled/styled-components-test'
 
```

Comment:
I think you might want these three:
- `@xstyled/styled-components` (direct match)
- `@xstyled/styled-components/test` (globbed match)
- `@xstyled/styled-components-test` (doesn't match)

Those were the originals, I guess. What was the motivation to change it?

Reply:
This plugin only checks the first import match than bails as far as I can tell. It only matches one and reordering the import causes a different one to be selected. I've decided to limit this test to the simple strict match and the other test can be for testing the picomatch behavior

Reply:
Sounds good!